README.md

```diff
@@ -53,7 +56,7 @@ cd kubeslice-controller
 ```console
 make docker-build
 ```
-### Running Local Image on Kind Clusters
+### Running Local Image on a Kind Clusters
```

Comment:
on a Kind Cluster  

a is not required for plurals

Reply:
done